apps/dg/components/case_card/case_card.js

```diff
@@ -385,20 +385,51 @@ DG.React.ready(function () {
             }.bind(this));
           },
 
+          /**
+           * -------------------Get first child case of selected case-----------------
+           */
+          getSelectedCaseFirstChildCaseID: function (iCollection) {
+              var tSelectedCaseFirstChildCaseID,
+                  tContext = this.props.context;
+              var tSelectedCase = tContext.getCollectionByID(iCollection.get('id')).getPath('casesController.selection').toArray();
+              if (tSelectedCase.length) {
+                tSelectedCaseFirstChildCaseID = tSelectedCase[0].children[0].id;
+              }
+              return tSelectedCaseFirstChildCaseID;
+          },
+
           /**
            *
            * @param iCollectionClient
            * @param iCaseIndex  {
            */
           moveToPreviousCase: function (iCollectionClient, iCaseIndex) {
+            var tPrevIndex = null;
+            var tNumCases = iCollectionClient.getPath('collection.cases').length;
+            if (SC.none(iCaseIndex) && iCollectionClient.collection.parent) {
```

Comment:
In SproutCore parlance, this should be `iCollectionClient.getPath('collection.parent')` analogous to the way cases are accessed above. The same pattern occurs below as well.